printrun/gcview.py

```diff
@@ -453,8 +453,7 @@ def __init__(self, parent, ID, title, build_dimensions, objects = None,
 
         h_sizer.Add(self.glpanel, 1, wx.EXPAND)
         h_sizer.Add(self.layerslider, 0, wx.EXPAND | wx.ALL, get_space('minor'))
-        self.glpanel.SetToolTip("Left-click to pan, right-click to move the view "
-                                "and shift + scroll to change the layer")
+        self.glpanel.SetToolTip(_("Click [Mouse-Right] to move the view, click [Mouse-Left]to pan the view, scroll with [Mouse Wheel] to zoom, and scroll with [Shift]+[Mouse Wheel] to change the layer."))
```

Comment:
Missing space between "[Mouse-Left]" and "to".

Reply:
Dito. And then update all translations :)

Reply:
Hi @rockstorm101,
All changes are done. Thanks for the review.

Reply:
Thank you for the work. Great stuff.